Tools/scripts/runcoptertest.py

```diff
@@ -35,24 +35,25 @@ def wait_time(mav, simtime):
         if t2 - t1 > simtime:
             break
 
-cmd = '../Tools/autotest/sim_vehicle.py -j4 -D'
+cmd = '../../Tools/autotest/sim_vehicle.py -D -S10'
 mavproxy = pexpect.spawn(cmd, logfile=sys.stdout, timeout=30)
 mavproxy.expect("Ready to FLY")
 
 mav = mavutil.mavlink_connection('127.0.0.1:14550')
 
 wait_time(mav, 2)
 mavproxy.send('mode GUIDED\n')
+mavproxy.send('param set SIM_SPEEDUP 5\n')
 wait_mode(mav, ['GUIDED'])
 mavproxy.expect('is using GPS')
 mavproxy.expect('is using GPS')
 mavproxy.send('arm throttle\n')
 mavproxy.expect('ARMED')
 mavproxy.send('mode AUTO\n')
 wait_mode(mav, ['AUTO'])
-wait_time(mav, 30)
+wait_time(mav, 15)
```

Comment:
Why did we reduce this time?

Reply:
this is just my little test script, 15s is closer to what we normally have between arming and takeoff. 30s was for when I was testing gps drift

Reply:
Okay Sounds good!